meetings/2025-04-09.md

```diff
@@ -0,0 +1,120 @@
+* MdJ: The patch PR came from SolidOS issues with CSS. 
+* eP: Do we agree that the CG's work does not give stability, it is all subject to change, and if you adopt early, you participate in the incubation. These are just CG drafts, we should communicate that clearly.
+* HZ: I want to point out that this is a dangerous message to send since it can hurt adoption.
+* JZ: I will bridge what Pavlik and Hadrian say. Wherever we diverge from WG we should make it explicit. This way people will know 
+* eP: I'm not sure I understand, the WG doesn't have a FPWD, it only has use cases and requirements. The CG should be tracking the WG, and refer to what they publish, when they publish it.
+* MdJ: I also thought about that, one one side we say we are incubating and open for experiments, but what is the effect on perception of Solid. Will in few years one protocol will be selected from it. Will people just say they will wait.
+* JZ: Was there a feedback from WG that they object direction we are taking?
+* MdJ: Aaron sent email giving ok to make changes as long as we don't publish 0.12. When we started doing it the resistance was more from SolidLab than the WG. We decided something in CG, published to solidcommunity.net and it looked that this is de-facto the standard. 
+* JZ: Is there no way to convince them to participate in CG?
+* eP: you say that CG decisions through pivot to solidcommunity.net become de-facto standards, but I don't see it that way, and it's misinforming people. What the CG does have zero standing until the WG adopts it. We can say we avoid making changes without technical commitment, but the most honest way is to say our work has no formal standing.
+* MdJ: I don't think we will reach conclusion or solution. Probably we can reach PR 711 and be aware of the dynamics.
+* T: ... unstable solidcommunity.net
+* JZ: There is already testing insteance.
+* MdJ: What does it mean that something gets merged in the spec repo. Someone runs into an issue using solidcommunity.net. Makes a PR to the specification repo, it gets merged, deployed to solidcommunity.net. If everyone runs their own server coherence gets even lower. Solidcommunity.net is used by SolidOS team to do demos etc. If we don't update that server we may don't bother updating the specification repo. 
+* eP: Even though the repo is called 'specification', we could include some more inline notes in cases where there is no full concensus.
+* MdJ: This way we can't have a test suite. We can't have compatible implementations. We already have some servers doing WAC, some ACP, some SAI others do UMA. 
+* MdJ: If Solid is a way to signal what can be done but there is no consistent way of achieving it. ...
+* JW: The CG is here to incubate specs. It should be a goal of consensus rather than specification divergance. For issue 711, that was particularly controversial, I don't think we will hit that many issues with this degree of controversy. 
```

Comment:
```suggestion
* JW: The CG is here to incubate specs. It should be a goal of consensus rather than specification divergence. For [PR 711](https://github.com/solid/specification/pull/711), that was particularly controversial. I don't think we will hit many issues with this degree of controversy. 
```